### PR TITLE
TE-887: Expose string props for OwnerLogin

### PR DESCRIPTION
**Pull request by stephenkr**
[Related YouTrack issue](https://youtrack.lodgify.net/issue/TE-887)

### What **one** thing does this PR do?
Exposes the string props for the `OwnerLogin`

__Preview of login form__
screen shot 2018-08-21 at 15 00 26

__Preview of forgot password form__
screen shot 2018-08-21 at 15 00 31

### CODE REVIEW

File: src/components/general-widgets/OwnerLogin/Readme.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
 ```jsx
 <OwnerLogin />
 ```
+
+#### Strings
+
+```jsx
+<OwnerLogin
+  emailLabel='Email Address'
```
Comment (Jonnygwi) [inline src/components/general-widgets/OwnerLogin/Readme.md]: Double quotes on this one please`emailLabel='Email Address'`

File: src/components/general-widgets/OwnerLogin/Readme.md [hunk 2]
```diff
@@ -1,3 +1,18 @@
 ```jsx
 <OwnerLogin />
 ```
+
+#### Strings
```
Comment (Jonnygwi) [inline src/components/general-widgets/OwnerLogin/Readme.md]: This should come under a `### Content` sub heading

File: src/components/general-widgets/OwnerLogin/Readme.md (hunk 1 above)
Comment (stephenkr) [reply src/components/general-widgets/OwnerLogin/Readme.md]: Ah ok, i'll fix it.

File: src/components/general-widgets/OwnerLogin/Readme.md (hunk 2 above)
Comment (stephenkr) [reply src/components/general-widgets/OwnerLogin/Readme.md]: Ok i'll update it.

File: src/components/general-widgets/OwnerLogin/utils/getForgotPasswordFormMarkup.js [hunk 3]
```diff
@@ -1,21 +1,32 @@
...
  * @return {Object}
  */
-export const getForgotPasswordFormMarkup = onForgotPasswordSubmit => (
-  <Modal trigger={<span>Forgot password</span>}>
+export const getForgotPasswordFormMarkup = (
+  onForgotPasswordSubmit,
+  forgotPasswordButtonText = SEND_RESET,
+  forgotPasswordEmailLabel = EMAIL,
+  forgotPasswordHeadingText = FORGOT_PASSWORD,
+  forgotPasswordModelTriggerText = FORGOT_PASSWORD
```
Comment (marcdavi-es) [inline src/components/general-widgets/OwnerLogin/utils/getForgotPasswordFormMarkup.js]: @stephenkr  No need for any of these default params - we're already covered by default props.

Comment (stephenkr) [reply src/components/general-widgets/OwnerLogin/utils/getForgotPasswordFormMarkup.js]: Oh ok, i had the mindset that the utility was self contained and didnt need to rely on the parent component.

I'll create a new PR to remove the default values.

Comment (stephenkr) [reply src/components/general-widgets/OwnerLogin/utils/getForgotPasswordFormMarkup.js]: The ticket https://youtrack.lodgify.net/issue/TE-921

**Pull request merged**